app/presenters/school_presenter.rb

```diff
@@ -3,6 +3,20 @@ def location
     [model.name, model.town, model.county].reject(&:blank?).join(', ')
   end
 
+  def school_size
+    if model.gias_data.present?
+      return model.gias_data['NumberOfPupils'] + ' pupils enrolled' if model.gias_data['NumberOfPupils'].present?
```

Comment:
I accept it's an edge case, but you should probably use `pluralize` here for those one-pupil schools :)

Reply:
I didn't even consider that edge case 😅 but that's a good point. I'll make sure to amend 😄